code/game/objects/structures/mineral_doors.dm

```diff
@@ -83,26 +83,47 @@
 		icon_state = "[base_icon_state][smoothing_flags ? "-[smoothing_junction]" : ""]"
 
 
-/obj/structure/mineral_door/attackby(obj/item/W, mob/living/user)
+/obj/structure/mineral_door/attackby(obj/item/attacking_item, mob/living/user)
 	. = ..()
 	if(.)
-		return
+		return TRUE
```

Comment:
don't do this. if there is already a return value, we return that. you're overriding it. While in this case that probably doesn't matter, its a bad habit.